Userland/Shell/Tests/builtin-test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+source $(dirname "$0")/test-commons.inc
+
+
+if not [ "$(type ls)" = "ls is /bin/ls" ] { fail "'type' on a binary not working" }
```

Comment:
This might not be the case on linux (we run the tests there), try `"ls is $(which ls)"` instead?

Reply:
Its still failing, any clue why?

Reply:
hmm...no idea, just run it again in the if body and let's see what the output is?

Reply:
Even better, is there a way to build the Shell on linux?

Reply:
Sure, just pass `-DBUILD_LAGOM=1` to your cmake invocation, and rebuild.
the shell is at `Build/$ARCH/Meta/Lagom/shell`

Reply:
https://github.com/alimpfard/serenity/commit/54ebae6261106b502bd8cdd7c3c707c1ef06deb9 should fix the capturing-builtin-output issue. you could either cherry-pick that commit, ignore the issue in this PR, or commit it yourself - whichever you're more comfortable with is fine.